Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -854,30 +1215,65 @@ function BuildTimeline {
     $Results = @();
 }
 
+# ===================================================================================================
+# Main
+# ===================================================================================================
+
+if (Get-Command -Name Get-Mailbox -ErrorAction SilentlyContinue) {
+    Write-Verbose "Validated Get-Mailbox"
+} else {
+    Write-Error "Get-Mailbox not found.  Please validate that you are running this script from an Exchange Management Shell and try again."
+    Write-Host "Look at Import-Module ExchangeOnlineManagement and Connect-ExchangeOnline."
+    exit;
+}
```

Comment:
Good point.   This is an EXO only script for now (as Get-CalDiaObjects is not on-prem yet).

Let me think how to Validate the passed in MB are EXO mailboxes.

For Now,  I will add some text that this will only work with EXO Mailboxes.